devel/quickfix/Portfile

```diff
@@ -15,61 +15,50 @@ description         FIX engine implementation
 long_description    QuickFIX is a full-featured open source FIX engine, \
                     currently compatible with the FIX 4.0-4.4 spec.
 
-homepage            http://www.quickfixengine.org/
+homepage            https://www.quickfixengine.org/
 master_sites        sourceforge:project/quickfix/quickfix/${version}
 worksrcdir          ${name}
 
-checksums           rmd160  369d5bf7044fa4c90d2d74e4470e50c0ad82951b \
-                    sha256  f6e8bdb004eaf45e50f63005b8c2611cb0afd42cf70f110f600c852aa572342d \
-                    size    21803784
+checksums           rmd160  63b491915df6e9b1809a97a4ba252bfc29219e26 \
+                    sha256  9842e71b5abc71f744aeacf65d8264cad2f360e32fa70d0c34f8b239384eb49c \
+                    size    46100428
 
 compiler.cxx_standard 2011
+supported_archs     x86_64
 
-depends_lib         port:libxml2
+depends_lib         port:libxml2 \
+                    port:openssl3
 
 patchfiles          patch-UnitTest++-Makefile.diff
 
 post-patch {
     reinplace "s|@CXX@|${configure.cxx}|g" ${worksrcpath}/UnitTest++/Makefile
     reinplace "s|@ARCHFLAGS@|[get_canonical_archflags cxx]|g" ${worksrcpath}/UnitTest++/Makefile
+    reinplace "s|`python3|`${prefix}/bin/python3.12|g" ${worksrcpath}/configure
 }
 
-configure.args      --with-java \
+configure.args      --without-java \
+                    --with-openssl=${prefix}/libexec/openssl3 \
```

Comment:
this perhaps is set also by the "openssl" PortGroup - please verify.

Reply:
It doesn't seem to set the `--with-openssl` but it does a good enough job of configuring cflags/ldflags that upstream's autotools figures it out from there. I will drop the explicit ${prefix} path